src/configure_transform.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+
+import os
+import re
+import sys
+
+define_re = re.compile("#define ([A-Z_]+) (.*)")
+
```

Comment:
Is `argparse` available on RHEL6?

Anyhow, no problem in sprucing it up as much as you'd like (I think you should be able to commit directly to the branch); I just didn't put much emphasis on this as it is a build script and not something that gets shipped.